operator/static/jquery.js

```diff
@@ -109,60 +121,42 @@ $(document).ready(function() {
-                interfaceIsActive = false;
-                logs.push(res.response);
-                $('#start-stop').removeClass('btn-danger').addClass('btn-primary').text('Send GO signal');
-                $('.log-window').append(`<p>${res.response}</p>`);
-            },
-            error: res => {
-                alert(res.responseJSON.error);
-            }
-        })     
-        :    
-        $.ajax('mission/start', {
-            data: JSON.stringify({access_token: accessToken}),
-            contentType: 'application/json',
-            type: 'POST',
-            success: res => {
-                interfaceIsActive = true;
-                logs.push(res.response);
-                $('#start-stop').removeClass('btn-primary').addClass('btn-danger').text('Send STOP signal'); 
-                $('.log-window').append(`<p>${res.response}</p>`);
-            },
-            error: res => {
-                alert(res.responseJSON.error);
-            }
-        });
-    });
-    
-    /**
-     * Reset button handler
-     */
-    $('#reset').click(function() {
-        const accessToken = $('#access-token').val();
-        if (accessToken === '') {
-            alert('Give an access token.');
-            return
+        $('#start-stop').prop('disabled', true);
+        $('#launch-exit').blur();
+        if(missionActive) {
+            $.ajax('mission/stop', {
+                data: JSON.stringify({access_token: accessToken,  sender: 'operator'}),
+                contentType: 'application/json',
+                type: 'POST',
+                success: res => {
+                    $('#start-stop').prop('disabled', false);
+                    missionActive = false;
+                    logs.push(res.response);
+                    $('#start-stop').removeClass('btn-danger').addClass('btn-primary').text('Send GO signal');
+                    $('.log-window').append(`<p>${res.response}</p>`);
+                },
+                error: res => {
+                    $('#start-stop').prop('disabled', false);
+                    alert(res.responseJSON.error);
+                }
+            })
+        } else {
+            $.ajax('mission/start', {
+                data: JSON.stringify({access_token: accessToken}),
+                contentType: 'application/json',
+                type: 'POST',
+                success: res => {
+                    $('#start-stop').prop('disabled', false);
+                    missionActive = true;
+                    logs.push(res.response);
+                    $('#start-stop').removeClass('btn-primary').addClass('btn-danger').text('Send STOP signal'); 
+                    $('.log-window').append(`<p>${res.response}</p>`);
+                },
+                error: res => {
+                    $('#start-stop').prop('disabled', false);
+                    alert(res.responseJSON.error);
+                }
+            });
         }
-    
-        $.ajax('mission/reset', {
```

Comment:
remove reset button because it won't be used during competition and it is only confusing.